### PR TITLE
Fix typo in ChainedScheduler docstring

### LINKED ISSUE #79720: ChainedScheduler documentation typoo
### 📚 The doc issue

```
Chains list of learning rate schedulers. It takes a list of chainable learning rate schedulers and performs consecutive step() functions **belong** to them by just one call.
```

### Suggest a potential alternative/fix

```
Chains list of learning rate schedulers. It takes a list of chainable learning rate schedulers and performs consecutive step() functions belonging to them by just one call.
```

### PR DESCRIPTION
**Pull request by bmsknight**
### Goal
Fixes https://github.com/pytorch/pytorch/issues/79720

### Approach 
replace `Chains list of learning rate schedulers. It takes a list of chainable learning rate schedulers and performs consecutive step() functions` **`belong`** `to them by just one call.` with `Chains list of learning rate schedulers. It takes a list of chainable learning rate schedulers and performs consecutive step() functions` **`belonging`** `to them by just one call.`

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Thanks for the fix
